src/plugin.ts

```diff
@@ -12,43 +12,37 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-import {
-  Application, IPlugin
-} from '@phosphor/application';
+import { Application, IPlugin } from '@lumino/application';
 
-import {
-  Widget
-} from '@phosphor/widgets';
+import { Widget } from '@lumino/widgets';
 
-import {
-  IJupyterWidgetRegistry
- } from '@jupyter-widgets/base';
+import { IJupyterWidgetRegistry } from '@jupyter-widgets/base';
 
 import * as widgetExports from './widget';
 
-import {
-  MODULE_NAME, MODULE_VERSION
-} from './version';
+import { MODULE_NAME, MODULE_VERSION } from './version';
 
-const EXTENSION_ID = 'displayWidget:plugin';
+const EXTENSION_ID = 'luxwidget:plugin';
 
 /**
- * The example plugin.
+ * The Lux plugin.
  */
-const examplePlugin: IPlugin<Application<Widget>, void> = {
+const luxPlugin: IPlugin<Application<Widget>, void> = ({
   id: EXTENSION_ID,
   requires: [IJupyterWidgetRegistry],
   activate: activateWidgetExtension,
-  autoStart: true
-};
-
-export default examplePlugin;
+  autoStart: true,
+});
```

Comment:
```
as unknown) as IPlugin<Application<Widget>, void>;
// the "as unknown as ..." typecast above is solely to support JupyterLab 1
// and 2 in the same codebase and should be removed when we migrate to Lumino.
```
To fix the type issue on installation, you can add this - but ideally we would not have this in the code. It was there when we used phosphor but lumino should take care of this issue.